examples/simple/topic.cs

```diff
@@ -0,0 +1,3 @@
+using LeanCode.Contracts;
+
+public class Topic : ITopic { }
```

Comment:
This should be blocked by analyzer as it does not make sense.

Reply:
Fair point, will add a rule for this.